gridstatus/miso.py

```diff
@@ -161,6 +164,90 @@ def _get_load_and_forecast_data(self, verbose=False):
         r = self._get_json(url, verbose=verbose)
         return r
 
+    forecast_cols = [
+        "Interval Start",
+        "Interval End",
+        "Publish Time",
+        "North",
+        "Central",
+        "South",
+        "MISO",
+    ]
+
+    @support_date_range(frequency="DAY_START")
+    def get_solar_forecast(self, date, verbose=False):
+        if date == "latest":
+            return self.get_solar_forecast(date="today", verbose=verbose)
+
+        return self._get_solar_and_wind_forecast_data(
+            date,
+            fuel="solar",
+            verbose=verbose,
+        )[self.forecast_cols]
+
+    @support_date_range(frequency="DAY_START")
+    def get_wind_forecast(self, date, verbose=False):
+        if date == "latest":
+            return self.get_wind_forecast(date="today", verbose=verbose)
+
+        return self._get_solar_and_wind_forecast_data(
+            date,
+            fuel="wind",
+            verbose=verbose,
+        )[self.forecast_cols]
+
+    def _get_solar_and_wind_forecast_data(self, date, fuel, verbose=False):
+        # Example url: https://docs.misoenergy.org/marketreports/20240327_mom.xlsx
+        url = f"https://docs.misoenergy.org/marketreports/{date.strftime('%Y%m%d')}_mom.xlsx"  # noqa
+
+        log(f"Downloading solar and wind forecast data from {url}", verbose)
+
+        try:
+            # Ignore the UserWarning from openpyxl about styles
+            warnings.filterwarnings("ignore", category=UserWarning, module="openpyxl")
+            excel_file = pd.ExcelFile(url, engine="openpyxl")
```

Comment:
there's a good change we are going to want to get other stuff from this excel like the outage data. i think you could make a separate report

`def get_mom_forecast_report`

Reply:
Good point. It might be easier to add those other datasets while I'm working with this file.

Reply:
It sounds like we do want to add outage data. I'll add that in a follow-up PR.